package.json

```diff
@@ -10,7 +10,7 @@
     "compile-sass": "sass --quiet-deps --no-source-map --load-path=node_modules/govuk-frontend --load-path=node_modules/@ministryofjustice/frontend --load-path=. ./assets/sass/application.sass:./assets/stylesheets/application.css --style compressed",
     "watch-ts": "tsc -w",
     "watch-views": "nodemon --watch server/views -e html,njk -x npm run copy-views",
-    "watch-node": "DEBUG=gov-starter-server* nodemon -r dotenv/config --watch dist/ dist/server.js | bunyan -o short",
+    "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ dist/server.js | bunyan -o short",
```

Comment:
@tpmcgowan not sure on this one, I did a pull before starting but didn't change this file 🤔

Reply:
We need the `-r dotenv/config` otherwise it won't load from `.env`. I added this recently when I incorporated changes from https://github.com/ministryofjustice/hmpps-template-typescript/pull/120. Just tried locally and I think this is the case (on a train with patchy signal so hard to be 100% sure 🚂!)

Reply:
Fixed!